src/xamarin-client/Adaptive.ReactiveTrader.Client.Domain/Transport/Wamp/ServiceObservableExtensions.cs

```diff
@@ -93,7 +93,7 @@ public static class ServiceObservableExtensions
                 disposables.Add(
                     source.Subscribe(innerSource =>
                     {
-                        IObservable<TValue> innerSourcePublished = innerSource.Replay(1).RefCount();
+                        IObservable<TValue> innerSourcePublished = innerSource.Publish().RefCount();
```

Comment:
this method is called `ToLastValueObservableDictionary` - if we change the behaviour, should we also change the method name?